sdk/src/transaction.rs

```diff
@@ -100,7 +100,11 @@ impl<P, Q> GenericInstruction<P, Q> {
     }
 }
 
-pub type Instruction = GenericInstruction<Pubkey, (Pubkey, bool)>;
+/// An account's pubkey and a Boolean that is true if an Instruciton
+/// requires a Transaction signature corresponding to this key.
+pub struct AccountMeta(pub Pubkey, pub bool);
```

Comment:
I personally prefer the full struct over tuples for public APIs:
```rust
pub struct AccountMeta {
  pubkey: Pubkey,
  is_signer: bool,
}
```

It makes the client code slightly more verbose:
```diff
-vec![AccountMeta(*from_id, true), AccountMeta(*to_id, false)],
+vec![AccountMeta{pubkey: *from_id, is_signer: true}, AccountMeta{pubkey: *to_id, is_signer: false}],
```
but getting "is_signer" right is super important so I feel that the extra effort is actually worth it in this case.   And you can still get the short form of `AccountMeta{pubkey, is_signer}` if your locals are named right.